app/uk/gov/hmrc/identitymanagementserviceproxy/controller/IdmsController.scala

```diff
@@ -16,7 +16,7 @@
 
 package uk.gov.hmrc.identitymanagementserviceproxy.controller
```

Comment:
we should take the opportunity to update the copyright where appropriate

Reply:
Ok

Reply:
Okay